roles/runtime/tasks/setup_aws.yml

```diff
@@ -60,4 +60,95 @@
       register: __dw_builds_async
       until: __dw_builds_async.finished
       retries: 120
-      delay: 30
\ No newline at end of file
+      delay: 30
+
+    - name: Retrieve CDP DW experiences
```

Comment:
If this is not used anywhere, there's no value in fetching it.

Reply:
This is being used in line 72 to fetch get the cluster id

Reply:
ah ok I missed that, then it's looking good to me :)